SoftLayer/CLI/email/edit.py

```diff
@@ -0,0 +1,32 @@
+"""Edit details of an Delivery email account."""
+# :license: MIT, see LICENSE for more details.
+
+import click
+
+from SoftLayer.CLI import environment
+from SoftLayer.CLI import exceptions
+from SoftLayer.managers.email import EmailManager
+
+
+@click.command()
+@click.argument('identifier')
+@click.option('--username', help="username account")
```

Comment:
```suggestion
@click.option('--username', help="Sets username for this account)
```

Can you actually change this option?